Gemfile

```diff
@@ -78,8 +80,9 @@ end
 
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> in views
-  gem 'web-console', '~> 2.0'
+  gem 'web-console', '>= 3.3.0'
   gem 'travis', '~> 1.8.0'
+  gem 'listen', '~> 3.1.5'
 end
```

Comment:
i was having all sorts of issues running the app locally and i think it's because when this app was upgraded to rails 5.2, i don't think we ran `rails app:update`. so i just ran that now and committed the parts of the diff that we want.

Reply:
there is still an issue with hot reloading when constants change, but we can figure that out next.